app/views/two_factor_authentication/shared/max_login_attempts_reached.html.erb

```diff
@@ -0,0 +1,18 @@
+<% lockout_time_in_words = decorated_user.lockout_time_remaining_in_words %>
+<% title t('titles.account_locked') %>
+
+<h1 class="h3 my0">
+  <%= t('titles.account_locked') %>
+</h1>
+<p>
+  <%= t('devise.two_factor_authentication.max_login_attempts_reached') %>
+</p>
+<p>
+  <%= t('devise.two_factor_authentication.please_try_again_html',
+  time_remaining: content_tag(:span, lockout_time_in_words, id: 'countdown')) %>
```

Comment:
Yeah that part does require one to know that the _html automatically calls `html_safe` on the output, but rails is already so full of magic what's one more 😬